postprocessor/analyzer.py

```diff
@@ -1,16 +1,23 @@
-from common.functions import log
-from postprocessor.quality import sanity_check, get_doc_quality
-from datetime import datetime
-import textstat
 import re
+import warnings
+
+import numpy
+import textstat
+import fasttext
 from ftlangdetect import detect
+from postprocessor.utils import log
+from postprocessor.quality import sanity_check, get_doc_quality
+
+fasttext.FastText.eprint = lambda x: None   # Suppress warnings from 'fasttext' library
+warnings.filterwarnings('ignore')           # Disable warnings from 'textstat' library
+
 
 class Analyzer(object):
 
     AVG_METRICS_DEF = ['avg_word_length', 'avg_sentence_length', 'noun_ratio', 'verb_ratio', 'adj_ratio', 'lexical_density', 'gunning_fog']
     MAX_TEXT_PART = 1024 * 1024 # Max text chunk part 
     CAMEL_CASE_PATTERN = re.compile(r"\b[a-ząęćłńóśżź]+[A-ZĄĘĆŁŃÓŚŻŹ]+[a-ząęćłńóśżź]+[a-ząęćłńóśżźA-ZĄĘĆŁŃÓŚŻŹ]*\b")
-    OBSOLETE_KEYS = ['length'] #A list of obsolete keys to remove from new meta
+    OBSOLETE_KEYS = ['length'] # A list of obsolete keys to remove from new meta

```

Comment:
Inline comments should be separated by at least 2 spaces:
`OBSOLETE_KEYS = ['length'] # A list...` -> `OBSOLETE_KEYS = ['length']  # A list` 

They are common to show us expected or sample output, but it is usually better not to use them for the readability reasons (a suggestion).